### PR TITLE
feat: add yomo completion spec

### PR DESCRIPTION
**Pull request by fanweixiao**
Hi Fig!

YoMo is an open source Streaming Serverless Framework to help developers with their Geo-distributed Applications. `yomo` is the CLI tool. Our community members love Fig so we want to Fig can help our developers too!

More about YoMo can be found here: https://github.com/yomorun/yomo

### CODE REVIEW

**Comment by fanweixiao** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/yomo.ts [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+          description: "Generate Rx code template",
+        },
+      ],
+    },
+    {
+      name: "build",
+      description: "Build a StreamFunction to WebAssembly",
+      args: {
+        name: ".go file",
+        description: "The .go file to build",
```
Comment (fedeci) [inline src/yomo.ts:24]: See https://fig.io/docs/reference/generator/prebuilt-generators#figautocomplete-generators
```suggestion
        description: "The .go file to build",
        generators: filepaths({ extensions: ["go"] }),
```

File: src/yomo.ts [hunk 2]
```diff
@@ -0,0 +1,73 @@
+const completionSpec: Fig.Spec = {
```
Comment (fedeci) [inline src/yomo.ts:1]:
```suggestion
import { filepaths } from "@fig/autocomplete-generators"

const completionSpec: Fig.Spec = {
```

File: src/yomo.ts [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+              name: "wasm",
+            },
+            {
+              name: "binary",
+            },
+          ],
+        },
+        {
+          name: ["-m", "--modfile"],
+          description: "Custom go.mod filepath",
```
Comment (fedeci) [inline src/yomo.ts:41]:
```suggestion
          description: "Custom go.mod filepath",
          args: {
            name: "module",
            generators: filepaths({ extensions: ["mod"] }),
          },
```

File: src/yomo.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+        },
+      ],
+    },
+    {
+      name: "run",
+      description: "Run a wasm stream function",
+      args: {
+        name: ".wasm file",
+        description: "The .wasm file to run",
```
Comment (fedeci) [inline src/yomo.ts:50]:
```suggestion
        description: "The .wasm file to run",
        generators: filepaths({ extensions: ["wasm"] }),
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks @fanweixiao! There are a couple improvements it would be great to have here, can you commit them?

File: src/yomo.ts (hunk 2 above)
Comment (fanweixiao) [reply src/yomo.ts:1]: oh, thanks @fedeci

**Comment by fanweixiao** [comment]
Thanks @fedeci

**Comment by fedeci** [comment]
I am merging the PR as soon as the checks are green!

**Comment by fedeci** [comment]
@fanweixiao, can you run `npm run lint:fix` and push the changes please?

**Comment by fanweixiao** [comment]
@fedeci I‘ve fixed this: 

```bash
$ npx prettier --check 'src/yomo.ts' --parser typescript
Checking formatting...
All matched files use Prettier code style!
```

**Pull request merged**